controller/controller.go

```diff
@@ -133,8 +133,8 @@ func New(config Config) (*Controller, error) {
 	return c, nil
 }
 
-func (c *Controller) Boot() {
-	ctx := context.Background()
+func (c *Controller) Boot(ctx context.Context) {
+	ctx = setLoggerCtxValue(ctx, loggerKeyController, c.name)
```

Comment:
This simply had to be moved to propagate the controller name into the informer. I would like to keep the dependency of the controller to the informer as it separates them clearly and keeps the controller configuration itself leaner. We would need to do more ugly configuration hacks in order to default or assume somebody wants to configure something for the informer they want to initiate indirectly. That would it not really make it better IMO.